public/Highways.php

```diff
@@ -28,6 +28,7 @@ function index() {
    * @param int $id The database ID of the highway you'd like to view.
    * @return [Highway] The highway requested.
    */
+  //Do NOT change the return statement to @return Highway [Highway] unless you want a bad day
```

Comment:
Hah! Bad PHPStorm! 
Commit early, commit often, remember? That way you can do a binary search across commits to find where it all went wrong. ("git bisect" is darned cool.)